sphinx/ext/doctest.py

```diff
@@ -362,11 +362,11 @@ def get_filename_for_node(self, node: Node, docname: str) -> str:
             filename = relpath(node.source, self.env.srcdir)\
                 .rsplit(':docstring of ', maxsplit=1)[0]
         except Exception:
-            filename = self.env.doc2path(docname, base=None)
+            filename = self.env.doc2path(docname, base=False)
```

Comment:
The `base` keyword is no longer needed.
```suggestion
            filename = self.env.doc2path(docname, False)
```

Reply:
done